Services/Simulation/ScriptInterpreter.cs

```diff
@@ -46,5 +50,40 @@ public class ScriptInterpreter : IScriptInterpreter
                     return result;
             }
         }
+
+        public static void AddMethodStatusProperty(Dictionary<string, object> state)
+        {
+            state[DEVICE_METHOD_STATUS] = "";
+        }
+
+        public static void EnableSensorSimulation(Dictionary<string, object> state)
+        {
+            state.Add(SENSOR_SIMULATION_ENABLED, true);
```

Comment:
should you check to see if it exists & set it if yes?  ...adding it if it doesn't